src/__mocks__/VideoCompressor.ts

```diff
@@ -0,0 +1,5 @@
+import { NativeModules } from 'react-native';
+
+NativeModules.VideoCompressor = {
```

Comment:
This makes sure that anywhere that the `VideoCompressor` native module is referred to in code won't crash a jest test.  I also included the `compress` function as a jest mock here, so that I can test if it was called properly in the test.  It can (and should!) be reused, if the code base is built up with more tests.